docs/UserGuide.md

```diff
@@ -286,19 +286,24 @@ excuted.
 
 Delete a useless floating task/event/deadline on TaskTracker.
 
-    del "TASK_NAME"
+    del TASK_UNIQUE_INDEX
 
-* Task name should be in a pair of quotation marks.
+* `TASK_UNIQUE_INDEX` will be different from each to each single task.
```

Comment:
> This kind of design will need to come with a system that reuses the IDs of previous tasks.

Yes, I agree. We could let the index starts from the beginning again once the index reaches the upper bond.